libs/application/templates/parental-leave/src/fields/Review/index.tsx

```diff
@@ -411,87 +613,205 @@ const Review: FC<ReviewScreenProps> = ({
-                          id="payments.union"
-                          options={unionOptions}
-                        />
-                      ) : (
-                        <Text>
-                          {getSelectOptionLabel(unionOptions, 'payments.union')}
-                        </Text>
-                      )}
-                    </GridColumn>
-                  </GridRow>
-                </>
+          {statefulSpousePersonalAllowance === YES &&
+            statefulSpouseUseAsMuchAsPossible === NO && (
+              <GridColumn span="5/12">
+                <DataValue
+                  label={formatMessage(
+                    parentalLeaveFormMessages.personalAllowance.allowanceUsage,
+                  )}
+                  value={`${statefulSpouseUsage ?? 0}%`}
+                />
+              </GridColumn>
+            )}
+        </GridRow>
+      </ReviewGroup>
+
+      <ReviewGroup
+        isEditable={editable}
+        editChildren={
+          <>
+            <Label marginBottom={3}>
+              {formatMessage(parentalLeaveFormMessages.selfEmployed.title)}
+            </Label>
+
+            <RadioController
+              id="employer.isSelfEmployed"
+              name="employer.isSelfEmployed"
+              defaultValue={isSelfEmployed}
+              split="1/2"
+              options={[
+                {
+                  label: formatMessage(
+                    parentalLeaveFormMessages.shared.yesOptionLabel,
+                  ),
+                  value: YES,
+                },
+                {
+                  label: formatMessage(
+                    parentalLeaveFormMessages.shared.noOptionLabel,
+                  ),
+                  value: NO,
+                },
+              ]}
+              onSelect={(s: string) => {
+                setStatefulSelfEmployed(s as Boolean)
+              }}
+            />
+
+            {statefulSelfEmployed === NO && (
+              <Input
+                id="employer.email"
+                name="employer.email"
+                label={formatMessage(parentalLeaveFormMessages.employer.email)}
+                ref={register({
+                  required: statefulSelfEmployed === NO,
+                })}
+              />
+            )}
+          </>
+        }
+      >
+        <GridRow>
+          <GridColumn span="5/12">
+            <RadioValue
+              label={formatMessage(
+                parentalLeaveFormMessages.selfEmployed.title,
+              )}
+              value={isSelfEmployed}
+            />
+          </GridColumn>
+
+          <GridColumn span="5/12">
+            {statefulSelfEmployed === NO && (
+              <DataValue
+                label={formatMessage(parentalLeaveFormMessages.employer.email)}
+                value={employerEmail}
+              />
+            )}
+          </GridColumn>
+        </GridRow>
+      </ReviewGroup>
+
+      <ReviewGroup isEditable={editable}>
```

Comment:
I assume this will not be editable so can probably just be `false`

Reply:
Good point 👍